help/en/releasenotes/current-draft-note.shtml

```diff
@@ -412,7 +412,8 @@
     <h3>LogixNG</h3>
         <a id="LogixNG" name="LogixNG"></a>
         <ul>
-          <li></li>
+          <li>Pressing the F8 key when the spash screen is visible disables all the LogixNGs.</li>
+          <li>Pressing the F9 key when the spash screen is visible prevents LogixNGs to be activated.</li>
```

Comment:
`prevents LogixNG activation` would work better.  Or `prevents LogixNG being activated`.

This is a LogixNG system state, not a state of each LogixNG item.

Reply:
Fixed.